src/components/navBar/ NavBar.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import {useState}  from 'react';
+import "./navBar.css";
+// import Burger from "../Burger/burger"
+const NavBar = () => {
+
+  const [showLinks,setShowLinks] = useState<boolean>(true)
+
+  return (
+    <section className="navbar-container">
```

Comment:
Navbar should always be inside a nav tag, screen readers might not understand that this is the navigation menu.
**Reference** - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav#usage_notes

Reply:
Thanks!